CMakeLists.txt

```diff
@@ -29,38 +33,54 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)
 set(CMAKE_AUTOUIC ON)
 
-if(CMAKE_VERSION VERSION_LESS "3.7.0")
-    set(CMAKE_INCLUDE_CURRENT_DIR ON)
-endif()
-
-set(RELEASE_NAME "Makou_Reactor")
-set(RELEASE_VERSION "1.8.2")
-set(GUI_TARGET "${RELEASE_NAME}")
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
-find_package(Qt5 COMPONENTS Widgets OpenGL LinguistTools REQUIRED)
+find_package(QT NAMES Qt5 COMPONENTS Widgets OpenGL LinguistTools REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets OpenGL LinguistTools REQUIRED)
 find_package(ff7tk REQUIRED NO_MODULE COMPONENTS ff7tk ff7tkUtils)
 find_package(ZLIB REQUIRED)
 
+file(GLOB_RECURSE TS_FILES "${RELEASE_NAME}_*.ts")
```

Comment:
Generate them in their own sub dir as seen here: https://github.com/sithlord48/ff7tk/tree/master/lang